execution/tvf/tumble_test.go

```diff
@@ -146,12 +150,12 @@ func TestTumble_Get(t *testing.T) {
 				windowLength: tt.fields.windowLength,
 				offset:       tt.fields.offset,
 			}
-			got, err := r.Get(tt.args.variables)
+			got, err := r.Get(ctx, tt.args.variables)
 			if (err != nil) != tt.wantErr {
 				t.Errorf("Tumble.Get() error = %v, wantErr %v", err, tt.wantErr)
 				return
 			}
-			eq, err := execution.AreStreamsEqual(got, tt.want)
+			eq, err := execution.AreStreamsEqual(context2.Background(), got, tt.want)
```

Comment:
<...> then whats the reason for using context2.Background() here?

Reply:
automatic refactor, fixed